README.md

```diff
@@ -3,6 +3,33 @@
 [![Build Status](https://wso2.org/jenkins/buildStatus/icon?job=products/micro-integrator)](https://wso2.org/jenkins/job/products/job/micro-integrator/)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
+# WSO2 Micro Integrator
+
+## Introduction
+
+In recent times we have witnessed the growth of approaches like Continuous Integration and Continuous Deployment (CI/CD), which has subsequently led to the rise of microservices. Microservices are designed to start up very quickly, perform a specific task, and then shut down.
+
+WSO2 Enterprise Integration (EI) is an open source integration platform and the Micro Integrator is a cloud native variant of WSO2 EI.
+
+The Micro Integrator is a lightweight integration framework built for developers who would like to create and run their integrations and integrate microservices. Micro Integrator provides developers the freedom to quickly develop and run their integrations in a Microservices Architecture (MSA) environment and to deploy them using CICD. So with the Micro Integrator, developers are able to develop and manage composite microservices. 
+
+Furthermore, since the startup time is a critical aspect when working with microservices, the Micro Integrator profile is designed with an improved startup time than the conventional WSO2 ESB or the ESB profile of WSO2 EI. Thus, with this improved startup time, the Micro Integrator profile becomes container-friendly for you to work with your microservices in a container-based architecture. The Micro Integrator works natively on the Kubernetes ecosystem and it makes deployment with Docker a simple process.
```

Comment:
Are we focusing too much on the startup time here? Startup time is just one aspect.

Reply:
Is there anything we are missing that we can add to this?

Reply:
My feeling is that we are giving too much focus to startup time. IMO startup time might not be the main reason why someone wants to use micro integrator. Maybe we can reorganize to reduce the focus on startup time.